packages/maskbook/src/plugins/FileService/Preview.tsx

```diff
@@ -47,7 +47,7 @@ const useStyles = makeStyles((theme) => ({
     },
 }))
 
-export const Preview: React.FC<Props> = ({ info }) => {
+export function Preview({ info }: { info: FileInfo }) {
```

Comment:
revert it.

Reply:
the function is better if possible to use. it is easier to analyze for the tools and can avoid TDZ problem

Reply:
TDZ?

Reply:
Temporary dead zone.